runtime-modules/proposals/codex/src/lib.rs

```diff
@@ -85,20 +87,22 @@ const MAX_SPENDING_PROPOSAL_VALUE: u32 = 5_000_000_u32;
 // Max validator count for the 'set validator count' proposal
 const MAX_VALIDATOR_COUNT: u32 = 100;
 
-// Data container struct to fix linter warning 'too many arguments for the function' for the
-// create_proposal() function.
-struct CreateProposalParameters<T: Trait> {
-    pub origin: T::Origin,
-    pub member_id: MemberId<T>,
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Encode, Decode, Debug, Default, Clone, PartialEq, Eq)]
```

Comment:
Please, move new structs to the `proposal_types` module and create comments for both structs and their fields. Also please, rename that module to just `types` and downgrade it to the file instead of the folder.